### ISSUE TITLE
DebugSettings.EnableFrameRateCounter = true in Desktop causes an AccessViolation

### ISSUE THREAD

**Issue by jtbrower** [microsoft/microsoft-ui-xaml#2835]
**Describe the bug**
Create a new WinUI Desktop application and set the DebugSettings.EnableFrameRateCounter = true in the application constructor or the OnLaunched and an access violation exception will occur when the application is run.

```
Exception thrown at 0x7A0D38FC (Microsoft.UI.Xaml.dll) in WinUI_InDesktop.exe: 0xC0000005: Access violation reading location 0x00000000.
```
**Steps to reproduce the bug**
1. In the attached solution, run the WinUiDesktop (Package) application.  Note that you will be able to safely step over enabling the frame rate counter but the application will crash shortly after.
2. For comparision, using the same solution, run the BlankAppUWP (Package).  Note that the application will not crash and it will show the framerate counter as expected.

**Expected behavior**
If by chance the DebugSettings.EnableFrameRateCounter setting cannot be used with Desktop applications, then a non-fatal exception message should be thrown.  If this feature is supported in Desktop applications then it has a bug.

**Screenshots**

![image](https://user-images.githubusercontent.com/3423706/86524360-71d19880-be3f-11ea-9a57-6664997583bd.png)

**Version Info**
.Net5.0 Preview 6

NuGet package version: 
```    <PackageReference Include="Microsoft.WinUI" Version="3.0.0-preview1.200515.3" />```  

**Note that I have been making WinUI work with Preview 6 by adding this additional direct package reference, a trick I learned from Scott Jones.  Although I have been able to accomplish quite a bit of Window related work without any crashes, it was worth mentioning.**

```<PackageReference Include="Microsoft.Windows.CsWinRT" Version="0.1.0-prerelease.200623.5" />```

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| May 2020 Update (19041)            | Yes  |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :----------------- | :--------------- |
| Desktop            | Yes |
| Xbox               |  |
| Surface Hub        |  |
| IoT                |  |

**Additional context**
I have attached a solution for easy reproduction of the problem. 
[WinUIDesktop.zip](https://github.com/microsoft/microsoft-ui-xaml/files/4874342/WinUIDesktop.zip)

**Comment by hshristov** [comment]
Also happens with Preview2

**Comment by MikeHillberg** [comment]
@RBrid Fyi

**Comment by RBrid** [comment]
Tracked by internal task 27649649 (and originally bug 27512832).

**Comment by aybe** [comment]
This is still happening as of today.

**Comment by myblindy** [comment]
Also today

**Comment by duncanmacmichael** [comment]
Fixed in WinAppSDK 1.4-experimental1 on 05/23/23; closing bug.

**Comment by codendone** [comment]
Fixed for the next 1.5 release.

**Comment by bpulliam** [comment]
Automation error. This is fixed in the 1.5 release.
